### DISCUSSION TITLE
Seperating the control from the menu

### DISCUSSION THREAD

**Discussion by cubecleveland** [JedWatson/react-select#4730] [Q&A]
Hello, is there any way to seperate the controle from the menu?
What im trying to achieve is to show the list somewhere always open where the slecetion is shown somewhere else....

**Comment by hermbo** [comment]
Do you mean using an external button to open the select dropdown?

**Comment by cubecleveland** [comment]
Not necessarily, basically trying to have athe search bx and the dropdown
in different locations..... and not necessarily adjacent...


CUBE Workspace
Corporate: 6240 Som Center Rd #130
Solon, OH 44139
T: 216.245.9629

**Reply by hermbo** [reply]
You'll need to be more clear and provide a better example because your request is confusing.
